assets/shop/entrypoint.js

```diff
@@ -1,2 +1,11 @@
+import { startStimulusApp } from '@symfony/stimulus-bridge';
+
+// Registers Stimulus controllers from controllers.json and in the controllers/ directory
+export const app = startStimulusApp(require.context(
+  '@symfony/stimulus-bridge/lazy-controller-loader!./controllers',
+  true,
+  /\.[jt]sx?$/
+));
+
```

Comment:
What about adding this content to a `bootstrap.js` file ?

Reply:
done too